### PR TITLE
Addressing zellic audit findings

### PR DESCRIPTION
**Pull request by 0xAlcibiades**

### CODE REVIEW

File: src/interfaces/ITokenURIGenerator.sol [hunk 1]
```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: BUSL 1.1
-pragma solidity 0.8.11;
+pragma solidity 0.8.16;
 
 import "./IOptionSettlementEngine.sol";
 
-/// @title A token URI geneartor for Claim NFTs
+/// @title A token URI generator for Claim NFTs
 /// @author 0xAlcibiades
```
Comment (neodaoist) [inline src/interfaces/ITokenURIGenerator.sol:7]: Update NatSpec authors to match [src/TokenURIGenerator.sol](https://github.com/valorem-labs-inc/valorem-core/pull/143/files#diff-25d286b3e5866a161b226fabd9b71e0002d8b28ec5b4e2215c929ce53e6eaca6)

Comment (0xAlcibiades) [reply src/interfaces/ITokenURIGenerator.sol:7]: I'm just going to remove duplicate offer info from interfaces.

**Comment by 0xAlcibiades** [comment]
Screen Shot 2023-01-02 at 19 24 14

**Review by neodaoist — APPROVED** [review]
LGTM 🚀🚀🚀

**Pull request merged**